src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -329,7 +329,20 @@ int mongoSubCacheItemInsert
 
   if (stringFilterP != NULL)
   {
-    cSubP->expression.stringFilter = *stringFilterP;
+    std::string errorString;
+
+    if (!cSubP->expression.stringFilter.fill(stringFilterP, &errorString))
+    {
```

Comment:
So, you propose to leave the if {}   with just a comment inside?
I'm not aganst it, it's a bit strange, but as it gives more info to the comment inside somehow I find it OK